homeassistant/components/rainmachine/config_flow.py

```diff
@@ -74,6 +74,12 @@ def __init__(self):
                 CONF_PASSWORD: 'invalid_credentials'
             })
 
+        # Since the config entry doesn't allow for configuration of SSL, make
+        # sure it's set:
+        if not user_input.get(CONF_SSL):
```

Comment:
Won't this always set `CONF_SSL` to `DEFAULT_SSL` since we're checking the value of the item and not membership? We still want to allow setting this option in config yaml.

Reply:
Great catch!